plugin/editorconfig.vim

```diff
@@ -197,6 +197,7 @@ function! s:GetFilenames(path, filename) " {{{1
 endfunction " }}}1
 
 function! s:UseConfigFiles() abort " Apply config to the current buffer {{{1
+    let b:editorconfig_invoked = 1
```

Comment:
How about `b:editorconfig_tried` instead?  (Related to a suggested name change below for the other variable.)

Reply:
Sounds good. Renamed.